unbound/metadata.csv

```diff
@@ -0,0 +1,82 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+unbound.thread.num.queries,count,,query,,The number of queries received by thread,1,unbound,thread.num.queries
+unbound.thread.num.queries_ip_ratelimited,count,,query,,The number of queries rate limited by thread,1,unbound,thread.num.queries_ip_ratelimited
+unbound.thread.num.cachehits,count,,query,,The number of queries that were successfully answered using a cache lookup,1,unbound,thread.num.cachehits
+unbound.thread.num.cachemiss,count,,query,,The number of queries that needed recursive processing,1,unbound,thread.num.cachemiss
+unbound.thread.num.dnscrypt.crypted,count,,query,,The number of queries that were encrypted and successfully decapsulated by dnscrypt,1,unbound,thread.num.dnscrypt.cert
+unbound.thread.num.dnscrypt.cert,count,,query,,The number of queries that were requesting dnscrypt certificates,1,unbound,thread.num.dnscrypt.cert
+unbound.thread.num.dnscrypt.cleartext,count,,query,,The number of queries received on dnscrypt port that were cleartext and not a request for certificates,1,unbound,thread.num.dnscrypt.cleartext
+unbound.thread.num.dnscrypt.malformed,count,,,,The number of requests that were neither cleartext nor valid dnscrypt messages,1,unbound,thread.num.dnscrypt.malformed
+unbound.thread.num.prefetch,count,,,,The number of cache prefetches performed,1,unbound,thread.num.prefetch
+unbound.thread.num.zero_ttl,count,,,,The number of replies with ttl zero because they served an expired cache entry,1,unbound,thread.num.zero_ttl
+unbound.thread.num.recursivereplies,count,,,,The number of replies sent to queries that needed recursive processing,1,unbound,thread.num.recursivereplies
+unbound.thread.requestlist.avg,gauge,,,,The average number of requests in the  internal  recursive  processing request list on insert of a new incoming recursive processing query,0,unbound,thread.requestlist.avg
+unbound.thread.requestlist.max,gauge,,,,Maximum size attained by the internal recursive processing request list,1,unbound,thread.requestlist.max
+unbound.thread.requestlist.overwritten,count,,,,The number of requests in the request list that were overwritten by newer entries,1,unbound,thread.requestlist.overwritten
+unbound.thread.requestlist.exceeded,gauge,,,, Queries that were dropped because the request list was full,1,unbound,thread.requestlist.exceeded
```

Comment:
```suggestion
unbound.thread.requestlist.exceeded,gauge,,query,, Queries that were dropped because the request list was full,1,unbound,thread.requestlist.exceeded
```